plugins/importexport/datacite/DataciteInfoSender.inc.php

```diff
@@ -137,8 +141,10 @@ function _getJournals() {
 			if ($doiPrefix) {
 				$journals[] =& $journal;
 			} else {
-				$this->notify(SCHEDULED_TASK_MESSAGE_TYPE_WARNING,
-					__('plugins.importexport.crossref.senderTask.warning.noDOIprefix', array('path' => $journal->getPath())));
+				$this->addExecutionLogEntry(
```

Comment:
Shouldn't this depend on whether the user is executing this from the web interface, or whether this is called via a cron/acron job?